gordo_components/server/views/base.py

```diff
@@ -237,6 +236,7 @@ class DownloadModel(Resource):
     @api.doc(
         description="Download model, loadable via gordo_components.serializer.loads"
     )
+    @server_utils.model_required
```

Comment:
Ok for now I guess, but this will cause the server to first load the model, then write it back again to a temp directory, which it again reads using the tar-stuff and sends over the network, instead of just taring the folder-structure it already has?

Reply:
This is, in effect, exactly how we had it before. Server would load it, but serialize the loaded model upon request. And this endpoint is called so little, the effectual difference would go unnoticed IMO, but alas, your wish is my command.

Reply:
I take that back, unless I misunderstand, to 'tar the directly it already has' would be to duplicate the logic we have for what _"serializer.loads"_ requires. If in the future we change `serializer.dumps` to be pickle-able, we'll _also_ have to remember that we need to update the server's `/download` logic. By depending on `serializer.dumps`  we know the two are compatible.